otx/cli/utils/tests.py

```diff
@@ -562,57 +562,70 @@ def otx_find_testing():
-        command_line = [
-            "otx",
-            "build",
-            "--task",
-            task,
-            "--workspace-root",
-            os.path.join(root, f"otx-workspace-{task}"),
-        ]
-        check_run(command_line)
+    command_line = [
+        "otx",
+        "build",
+        "--task",
+        task,
+        "--workspace-root",
+        os.path.join(root, f"otx-workspace-{task}"),
+    ]
+    check_run(command_line)
 
-    for task, backbone in args.items():
-        task_workspace = os.path.join(root, f"otx-workspace-{task}")
-        # Build Backbone.yaml from backbone type
-        command_line = [
-            "otx",
-            "build",
-            "--backbone",
-            backbone,
-            "--workspace-root",
-            task_workspace,
-            "--save-backbone-to",
-            os.path.join(task_workspace, "backbone.yaml"),
-        ]
-        check_run(command_line)
-        assert os.path.exists(os.path.join(task_workspace, "backbone.yaml"))
 
-        # Build model.py from backbone.yaml
-        command_line = [
-            "otx",
-            "build",
-            "--model",
-            os.path.join(task_workspace, "model.py"),
-            "--backbone",
-            os.path.join(task_workspace, "backbone.yaml"),
-            "--workspace-root",
-            task_workspace,
-        ]
-        check_run(command_line)
+def otx_build_backbone_testing(root, backbone_args):
+    """Build backbone & Update model testing.
 
-        # Build model.py from backbone type
-        command_line = [
-            "otx",
-            "build",
-            "--model",
-            os.path.join(task_workspace, "model.py"),
-            "--backbone",
-            backbone,
-            "--workspace-root",
-            task_workspace,
-        ]
-        check_run(command_line)
+    Build each backbone to create backbone.yaml into workspace,
+    build for the default model for the task,
+    and even test updating the model config.
+    This is done on the premise that the otx_workspace
+    has been created well through otx_build_task_testing.
+    """
+    task, backbone = backbone_args
+    task_workspace = os.path.join(root, f"otx-workspace-{task}")
+    # Build Backbone.yaml from backbone type
+    command_line = [
+        "otx",
+        "build",
+        "--backbone",
+        backbone,
+        "--workspace-root",
+        task_workspace,
+        "--save-backbone-to",
+        os.path.join(task_workspace, "backbone.yaml"),
+    ]
+    check_run(command_line)
+    assert os.path.exists(os.path.join(task_workspace, "backbone.yaml"))
+
+    # Build model.py from backbone.yaml
+    command_line = [
+        "otx",
+        "build",
+        "--model",
+        os.path.join(task_workspace, "model.py"),
+        "--backbone",
+        os.path.join(task_workspace, "backbone.yaml"),
+        "--workspace-root",
+        task_workspace,
+    ]
+    check_run(command_line)
+
```

Comment:
how about adding some asserts code to check whether code work as expected?

Reply:
Added this.